app/Http/Requests/UpdateUserRequest.php

```diff
@@ -48,13 +49,18 @@ public function rules(): array
                 'max:127',
             ],
             'phone' => [
-                'max:15',
+                'digits_between:10,15',
             ],
             'emergency_contact_name' => [
+                'nullable',
```

Comment:
Do we really want to make this optional? Then we have to go back and make sure people filled it out before they go on a trip.

Reply:
Most people that pay dues do not travel.